aten/src/ATen/native/cudnn/Conv_v8.cpp

```diff
@@ -180,23 +181,93 @@ struct CacheKeyFusedWrapper : ParamsWrapper<CacheKeyFused> {
   }
 };
 
+static int getLRUCacheLimit() {
+  constexpr int DEFAULT_LIMIT = 10000; // roughly corresponds to 2GiB assuming 200KiB per ExecutionPlan
+  static int limit = [&] {
+    const char * val = getenv("TORCH_CUDNN_V8_API_LRU_CACHE_LIMIT");
+    if (!val) {
+       return DEFAULT_LIMIT;
+    }
+    try {
+      return std::stoi(val);
+    } catch(std::invalid_argument const& e) {
+      TORCH_WARN("invalid TORCH_CUDNN_V8_API_LRU_CACHE_LIMIT,",
+               " using default LRU cache limit of ", limit, " entries.");
```

Comment:
As `limit` is undefined at this point (and in general I wonder what will happen in this case, deadlock perhaps?)
```suggestion
               " using default LRU cache limit of ", DEFAULT_LIMIT, " entries.");
```